### PR TITLE
restic-browser 0.3.1 (new cask)

### PR DESCRIPTION
**Pull request by SoCuul**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/r/restic-browser.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask "restic-browser" do
+  version "0.3.1"
+  sha256 "30fc1daebc2705d15f85803f1aee9a37ff2b5d99278c3ee4db9482e427a04c2c"
+
+  url "https://github.com/emuell/restic-browser/releases/download/v#{version}/Restic-Browser-v#{version}-macOS.zip"
+  name "restic-browser"
```
Comment (khipp) [inline Casks/r/restic-browser.rb:6]:
```suggestion
  name "Restic Browser"
```

File: Casks/r/restic-browser.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  desc "GUI to browse and restore restic backup repositories"
+  homepage "https://github.com/emuell/restic-browser"
+
+  livecheck do
+    url :url
+    strategy :github_releases
+  end
+
+  depends_on macos: ">= :high_sierra"
```
Comment (khipp) [inline Casks/r/restic-browser.rb:15]:
```suggestion
  depends_on formula: "restic"
  depends_on macos: ">= :high_sierra"
```

**Review by khipp — APPROVED** [review]
Thank you, @SoCuul!

File: Casks/r/restic-browser.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "30fc1daebc2705d15f85803f1aee9a37ff2b5d99278c3ee4db9482e427a04c2c"
+
+  url "https://github.com/emuell/restic-browser/releases/download/v#{version}/Restic-Browser-v#{version}-macOS.zip"
+  name "Restic Browser"
+  desc "GUI to browse and restore restic backup repositories"
+  homepage "https://github.com/emuell/restic-browser"
+
+  livecheck do
+    url :url
+    strategy :github_releases
```
Comment (bevanjkay) [inline Casks/r/restic-browser.rb:12]:
```suggestion
    strategy :github_latest

**Review by daeho-ro — APPROVED** [review]

**Pull request merged**